charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/job-patchAPIService.yaml

```diff
@@ -3,50 +3,58 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice-patch
+  name: {{ template "common.naming.fullname" . }}-apiservice-patch
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
   labels:
-    app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 4 }}
+    app: {{ template "common.naming.name" $ }}
+{{ include "common.labels" $ | indent 4 }}
 spec:
   template:
     metadata:
-      name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice-patch
+      name: {{ template "common.naming.fullname" . }}-apiservice-patch
       labels:
-        app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 8 }}
+        app: {{ template "common.naming.name" $ }}
+{{ include "common.labels" $ | indent 8 }}
     spec:
       {{- if .Values.apiServicePatchJob.image.pullSecrets }}
       imagePullSecrets:
         {{- toYaml .Values.apiServicePatchJob.image.pullSecrets | nindent 8 }}
       {{- end }}
```

Comment:
Is there any issue that does not allow us to use a global pull secrets here?

Reply:
Yes, I found it weird that we had 2 separated ones, maybe @gsanchezgavier or @paologallinaharbur can clarify why it was thought having 2 different pullsecrets.

Reply:
Fixed in last commits